start.R

```diff
@@ -344,6 +347,7 @@ if (any(c("--reprepare", "--restart") %in% argv)) {
     cfg$remind_folder <- getwd()                      # overwrite remind_folder: run to be restarted may have been moved from other repository
     cfg$results_folder <- paste0("output/",outputdir) # overwrite results_folder in cfg with name of the folder the user wants to restart, because user might have renamed the folder before restarting
     save(cfg,file=paste0("output/",outputdir,"/config.Rdata"))
+    startedRuns <<- startedRuns + 1
```

Comment:
Do we really need `<<-` here? This operator violates the usual R scoping rules so it should be avoided if possible. This line is not in a function, so replacing `<<-` with the normal `<-` should just work.

Reply:
yes, I tested it. There are two other situations below and also in start_bundle_coupled.R where a similar call is indeed in a function – if there is a better solution, let me know.

Reply:
A function should never change values in the calling/outer scope, but it can use the return value: Then the calling scope should analyze the return value and change variables in the calling scope accordingly.